add edit option and rofi screenshot applet (#26)
hyprwm/contrib: README.md

Contributions are welcome! Please open a PR with your utility and describe how
it helps the Hypr ecosystem. Also provide a screen recording if possible,
demonstrating its usage.

# Changelog

> Add 'edit' to Grimpblast option read the [man page](grimblast/grimblast.1.scd) for more information
> 
> Copy of applet screenshot script for rofi

Reviewer: ```suggestion
Added the `edit` option to Grimblast. read the [man page](grimblast/grimblast.1.scd) for more information.
A rofi script is also included.
```
